hanabi_lib/canonical_encoders.cc

```diff
@@ -702,6 +738,16 @@ std::vector<float> CanonicalObservationEncoder::Encode(
     const std::vector<int>& color_permute,
     const std::vector<int>& inv_color_permute,
     bool hide_action) const {
+
+  if (using_joint_obs_) {
```

Comment:
There is no need to call EncodeJointFivePlayers into this Encode function. We can just call `encoder. EncodeJointFivePlayers` in the `hanabi_env.cc`